src/Auth/SeedDatabase.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Codidact.Infrastructure.Identity;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.DependencyInjection;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Codidact.Auth
+{
+    public static class SeedDatabase
+    {
+        public async static Task InitializeIdentityUsers(this IApplicationBuilder app)
+        {
+            var toCreateApplicationUsers = new List<ApplicationUser>
+            {
+                new ApplicationUser
+                {
+                    UserName= "admin",
+                    Email = "admin@codidact.com",
+                }
+            };
+            using var serviceScope = app.ApplicationServices.GetRequiredService<IServiceScopeFactory>().CreateScope();
+            using var userManager = serviceScope.ServiceProvider.GetService<UserManager<ApplicationUser>>();
+            foreach (var entity in toCreateApplicationUsers)
+            {
+                var exists = await userManager.FindByNameAsync(entity.UserName);
+                if (exists == null)
+                {
+                    await userManager.CreateAsync(entity, "Aa123456!");
```

Comment:
That is a good idea

Reply:
Fixed in https://github.com/codidact/core/pull/29/commits/855206b88160c5281b1922dd75e696943d432a35
But its just a simple GUID. The person running an instance should be reasonable and change his password.
I don't think its a good idea to hold the code that generates passwords in the code base because then people will think its safe and won't be encouraged to change their passwords.

Reply:
Ended up adding a generate password method in https://github.com/codidact/core/pull/29/commits/285787fc53560d31c9fc5c407580d3f45d1f0d4c because GUID passwords werent up to standards of the framework.